### PR TITLE
Development

### PR DESCRIPTION
**Pull request by danielkryska**

### CODE REVIEW

**Comment by Marcelinna** [comment]
Master - no login possible
image


PR 
image

**Comment by Marcelinna** [comment]
When the user click on the button "see all" under "Software", he is redirected to the Publication
image

**Comment by Marcelinna** [comment]
When user click on the button "clear result", he is redirected to "All catalogues"
PR 
image

Master
image

**Comment by Marcelinna** [comment]
Data or Dataset?
image

**Comment by Marcelinna** [comment]
this is an intentional removal from PR ?

PR
image

Master
image

**Comment by Marcelinna** [comment]
this is an intentional removal of "Subject" from the Publications filter?
PR 
image

Master
image

**Comment by Marcelinna** [comment]
Click on the Traning redirect to extermal source

PR
image
Master
image

**Comment by Marcelinna** [comment]
PR
image
Master
image

**Comment by Marcelinna** [comment]
There are other collections in the publications search results
image

**Comment by Marcelinna** [comment]
For training are new filters with no result
image
image

**Comment by Marcelinna** [comment]
Click on All catalog breadcrumbs when you are on Software, Services or Tranings
image
image

**Review by Marcelinna — APPROVED** [review]

File: ui/apps/ui/src/app/components/search-input/search-input.component.ts [hunk 1]
```diff
@@ -0,0 +1,215 @@
...
+  ) {}
+
+  ngOnInit() {
+    // IMPORTANT!! NEEDED FOR MULTI-PAGES COMPONENTS
+    this._customRouter._router = this._router;
+    // // // // //
+
+    this._customRouter.q$
+      .pipe(
+        untilDestroyed(this),
```
Comment (michal-szostak) [inline ui/apps/ui/src/app/components/search-input/search-input.component.ts:175]: `untilDestroyed(this)` missing

File: ui/apps/ui/src/app/components/search-input/search-input.component.ts [hunk 2]
```diff
@@ -0,0 +1,215 @@
...
+
+  constructor(
+    private _customRouter: CustomRouter,
+    private _router: Router,
+    private _searchInputService: SearchInputService
+  ) {}
+
+  ngOnInit() {
+    // IMPORTANT!! NEEDED FOR MULTI-PAGES COMPONENTS
+    this._customRouter._router = this._router;
```
Comment (michal-szostak) [inline ui/apps/ui/src/app/components/search-input/search-input.component.ts:170]: This is very hacky...

**Review by michal-szostak — APPROVED** [review]

**Pull request merged**